### PR TITLE
adding blog post about websub

### PR DESCRIPTION
**Pull request by julien51**
# Description

All in the title!

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-protocol-com/blog/websub.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+subTitle: Enabling subscriptions to new locks and new keys!
+authorName: Julien Genestoux
+publishDate: Feb 22, 2022
+description: WebSub is a W3C protocol that provides a common mechanism for communication between publishers of any kind of Web content and their subscribers, based on HTTP web hooks.
+image: /images/blog/websub.png
+---
+
+As developers integrate the Unlock Protocol in their applications, we are getting many feature requests to add APIs and tools that would make their lives easier.
+
+![Webflow](/images/blog/websub.png)
```
Comment (julien51) [inline unlock-protocol-com/blog/websub.md:12]:
```suggestion
![Websub](/images/blog/websub.png)
```

File: unlock-protocol-com/blog/websub.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+image: /images/blog/websub.png
+---
+
+As developers integrate the Unlock Protocol in their applications, we are getting many feature requests to add APIs and tools that would make their lives easier.
+
+![Websub](/images/blog/websub.png)
+
+Today, we're excited to announce our [WebSub endpoints](https://docs.unlock-protocol.com/unlock/developers/locksmith/webhooks)!
+
+WebSub is a [W3C protocol](https://www.w3.org/TR/websub/) that provides a common mechanism for communication between publishers of any kind of Web content and their subscribers, based on HTTP **web hooks**.
```
Comment (desertrat10) [inline unlock-protocol-com/blog/websub.md:16]: WebSub [W3C protocol](https://www.w3.org/TR/websub/) provides a common mechanism for communication between publishers of any kind of Web content and their subscribers, based on HTTP **web hooks**.

File: unlock-protocol-com/blog/websub.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+This is a very powerful mechanism to enable Publish/Subscribe on any kind of HTTP resource! We added this as a way for applications to subscribe to changes in the Unlock: applications can get notified when new locks are deployed or when new keys are created.
```
Comment (desertrat10) [inline unlock-protocol-com/blog/websub.md:18]: WebSub is a powerful mechanism to enable Publish/Subscribe on a diversity of HTTP resources! We added this feature so applications may subscribe to changes in Unlock and get notified when new locks are deployed or when new keys are created.

File: unlock-protocol-com/blog/websub.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+Our team used this new API in order to [create a Discord bot](https://github.com/unlock-protocol/websub-discord) that notifies us when a new lock has been deployed or when a new key has been purchased.
+
+We'd love to see how your application ends up using that new capabilities!
```
Comment (desertrat10) [inline unlock-protocol-com/blog/websub.md:22]: We'd love to see how your application deploys this powerful new feature!

**Review by desertrat10 — COMMENTED** [review]
Looks great! I just made some minor tweaks to remove a few "this" and make a couple sentences flow better. But looks solid! ;)

**Pull request merged**